src/Themes/NexusMods.Themes.NexusFluentDark/Styles/UserControls/CollectionCards/CollectionCardStyles.axaml

```diff
@@ -0,0 +1,223 @@
+        xmlns:collections="clr-namespace:NexusMods.App.UI.Pages.LibraryPage.Collections;assembly=NexusMods.App.UI">
+
+    <Design.PreviewWith>
+        <Border Classes="Transparent" Width="600" Height="380">
+            <panels:FlexPanel
+                ColumnSpacing="8"
+                Wrap="Wrap"
+                RowSpacing="8"
+                Margin="30">
+                <panels:FlexPanel.Resources>
+                    <collections:CollectionCardDesignViewModel x:Key="LoadoutCardDesignViewModel" />
+                </panels:FlexPanel.Resources>
+                <collections:CollectionCardView DataContext="{StaticResource LoadoutCardDesignViewModel}" />
+            </panels:FlexPanel>
+        </Border>
+
+    </Design.PreviewWith>
+
+    <Style Selector="collections|CollectionCardView">
+        <Setter Property="Background" Value="{StaticResource SurfaceTranslucentLow}" />
+        <Setter Property="CornerRadius" Value="{StaticResource Rounded}" />
+        <Setter Property="Width" Value="508" />
+        <Setter Property="Height" Value="288" />
+        <Setter Property="ClipToBounds" Value="False" />
+        
+        <Style Selector="^ Border#CardCollectionBorder">
+            <Setter Property="BoxShadow" Value="{StaticResource DropShadowXS}" />
+        </Style>
+
+        <!-- top row -->
+        <Style Selector="^ panels|FlexPanel#CardCollection">
+            <Setter Property="Direction" Value="Column" />
+
+            <Style Selector="^ Border#ContainerBorder">
+                <Setter Property="Padding" Value="12, 12, 12, 0" />
+
+                <Style Selector="^ panels|FlexPanel#Container">
+                    <Setter Property="AlignItems" Value="FlexStart" />
+                    <Setter Property="panels:Flex.AlignSelf" Value="Stretch" />
+                    <Setter Property="Direction" Value="Row" />
+
+                    <!-- left column -->
+                    <Style Selector="^ Border#TileImageBorder">
+                        <Setter Property="Background" Value="DarkRed" />
+                        <Setter Property="CornerRadius" Value="{StaticResource Rounded}" />
+                        <Setter Property="ClipToBounds" Value="True" />
+                        <!-- NOTE(insomnious): not sure if we fixed sizes on both elements, but figma does -->
+                        <Setter Property="Width" Value="144" />
+                        <Setter Property="Height" Value="180" />
+                        <Style Selector="^ Image#TileImage">
+                            <Setter Property="Width" Value="144" />
+                            <Setter Property="Height" Value="180" />
+                        </Style>
+                    </Style>
+
+                    <!-- right column -->
+                    <Style Selector="^ panels|FlexPanel#TitleContainer">
+
+                        <!-- <Setter Property="Background" Value="DarkBlue" /> -->
```

Comment:
Debating whether to keep these or not, they are actually pretty nice to quickly locate the selected element in the preview, so leaning towards keeping these

Reply:
I'd like to keep for now, we will be doing more passes to these styles as we are far from final